package.json

```diff
@@ -57,9 +58,6 @@
     "typechain": "^8.1.1",
     "typescript": "~4.9.3"
   },
-  "dependencies": {
```

Comment:
Why is this being removed? Pretty sure this is necessary to be able to build the typed packages being deployed.

Reply:
I made a selection error when I removed the hardhat and used forge for the ast